synapse/tests/test_lib_aha.py

```diff
@@ -418,3 +421,61 @@ async def test_lib_aha_bootstrap(self):
 
                     async with await s_telepath.openurl(f'ssl://root@127.0.0.1:{port}?hostname=aha.do.vertex.link') as proxy:
                         await proxy.getCellInfo()
+
+    async def test_lib_aha_provision(self):
+
+        with self.getTestDir() as dirn:
+
+            conf = {
+                'aha:name': 'aha.loop.vertex.link',
```

Comment:
We should use the `skipIfNoInternet()` helper here to at least grant a optional out for disabling the test for running it offline.

Reply:
I'll just change the `dmon:listen` to 0.0.0.0 and it wont actually require Internet access.